server/sonar-main/src/main/java/org/sonar/application/command/CommandFactoryImpl.java

```diff
@@ -67,27 +67,72 @@ public CommandFactoryImpl(Props props, File tempDir, System2 system2) {
   }
 
   @Override
-  public EsCommand createEsCommand() {
-    EsFileSystem esFileSystem = new EsFileSystem(props);
-    if (!esFileSystem.getExecutable().exists()) {
+  public AbstractCommand<?> createEsCommand() {
+    // FIXME reenable
+    //if (IS_OS_WINDOWS) {
+      return createEsCommandForWindows();
+    //}
+    //return createEsCommandForNonWindows();
+  }
+
+  private ExternalCommand createEsCommandForNonWindows() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "createEsCommandForNonWindows" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUnusedPrivateMethod)